docs/gettingstarted/index.md

```diff
@@ -53,13 +53,23 @@ Edit the `build.gradle` file to customize how your mod is built (e.g., file name
 
 #### Mod Id Replacement
 
-Replace all occurrences of `examplemod`, including [`mods.toml` and the main mod file][modfiles] with the mod id of your mod. This also includes changing the name of the file you build by setting `archivesBaseName` (this is typically set to your mod id).
+Replace all occurrences of `examplemod`, including [`mods.toml` and the main mod file][modfiles] with the mod id of your mod. This also includes changing the name of the file you build by setting `base.archivesName` (this is typically set to your mod id).
 
 ```gradle
 // In some build.gradle
-archivesBaseName = 'mymod'
+base.archivesName = 'mymod'
 ```
 
+!!! note
+    The Forge MDK currently uses `archivesBaseName` to set the artifact name instead of `base.archivesName`. We recommend using `base.archivesName` instead as `archivesBaseName` is deprecated for removal in Gradle 9, which a future version of ForgeGradle will support.
+
+    You can still use `archivesBaseName` by setting the following:
+
+    ```gradle
+    // In some build.gradle
+    base.archivesName = 'mymod'
+    ```
```

Comment:
I was more thinking on doing this change when we change the MDK as well, but it works too. :shipit: